kqueen_ui/config/dev.py

```diff
@@ -10,3 +10,15 @@ class Config(BaseConfig):
     SECURITY_PASSWORD_SALT = 'secret_salt'
 
     ENABLE_PUBLIC_REGISTRATION = True
+
+    # Authentication choices
+    AUTH_OPTIONS = {
+        'local': {
+            'label': 'Local',
+            'notify': True
```

Comment:
I didn't include it in prod and demo, because I don't currently expect LDAP server to be available on demo or in "production" (thats only the demo public deployment right now). I included it in tests, because I want the specific parts of code to be present during tests and I included it in dev basically just to get your feedback on it